src/radha/Assignment14/CalcScanner.java

```diff
@@ -0,0 +1,68 @@
+package radha.Assignment14;
+
+import java.util.Scanner;
+
+public class CalcScanner {
+	int add(){
+		Scanner scanner = new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method